documentation/guidance/before-starting.md

```diff
@@ -64,33 +69,21 @@ There are no registration fees or costs associated with publishing your grants d
 
 Preparing 360Giving data does require a commitment of resources from the publisher in terms of staff time, especially when publishing your data for the first time. 
 
-There can however be costs related to consultancy or support fees to get a grants management system set up to export 360Giving data. 
+However, there can be costs related to consultancy or support fees to set up a grants management system to export 360Giving data. 
 
 #### 360Giving Helpdesk
-Our Helpdesk provides pro-bono support to help funders navigate the steps to publish their grant data using the 360Giving Data Standard. We provide guidance and direct support to help streamline the process as far as possible.
-
-You can contact 360Giving Helpdesk via <support@threesixtygiving.org>. Visit our website to find out other ways to get <a href="https://www.threesixtygiving.org/publishing" target="_blank">support with publishing</a>.
-
-## Who publishes 360Giving data
-Any type of funder can publish their grants data in the 360Giving Data Standard. Over 250 funders share 360Giving data. They range in size from large organisations with many staff to small trustee-run trusts and include family, corporate and community foundations, central, devolved and local governments, lottery distributors and livery companies, as well as benevolent charities and grantmakers to individuals.
-
-The majority of funders sharing 360Giving data make grants to recipients based in the UK, however it is also possible to use the 360Giving Data Standard to share data about international grants.
+Our Helpdesk provides pro-bono support to help funders navigate the steps to publish their grants data using the 360Giving Data Standard. We provide guidance and direct support to help streamline the process as far as possible.
 
-Although initially designed for sharing data about grants to organisations, an update to the 360Giving Data Standard in 2022 has enabled the publication of grants to individuals.
-
-View our <a href="https://www.threesixtygiving.org/data/explore-the-data" target="_blank">visualisations</a> to find out more about the different types of funders publishing 360Giving data.
-
-### Why publish 360Giving data?
-Find out more about <a href="https://www.threesixtygiving.org/support/why-publish-grants-data/" target="_blank">the benefits of publishing 360Giving data.</a> 
+You can contact 360Giving Helpdesk via <support@threesixtygiving.org>. Visit our website to find out other ways to <a href="https://www.360giving.org/support/?event_type%5B%5D=publish-grants-data&sort-by=date" target="_blank"> get support with publishing</a>.
```

Comment:
https://www.360giving.org/support/?event_type%5B%5D=publish-grants-data&sort-by=date This link results in a 404 (page not found). 

Contact us > [Attend our Publishing workshops](https://www.360giving.org/support/?event_type%5B%5D=publish-grants-data&sort-by=date) also links to the broken link. 

``` suggestion
You can contact 360Giving Helpdesk via <support@threesixtygiving.org>. Visit our website to find out other ways to <a href="https://www.360giving.org/support/?event_type%5B%5D=intro-to-publishing&event_type%5B%5D=grants-to-inds&sort-by=date" target="_blank"> get support with publishing</a>.